src/toM3u8.js

```diff
@@ -237,6 +229,13 @@ export const formatVideoPlaylist = ({ attributes, segments, sidx }) => {
   return playlist;
 };
 
+const videoOnly = ({ attributes }) =>
```

Comment:
We defined these in a function before, needlessly creating these functions on every mpd parse isn't something we should do.